.github/workflows/build-and-deploy.yml

```diff
@@ -109,4 +109,4 @@ jobs:
         with:
           header: staging url
           recreate: true
-          message: Deployed staging instance to https://staging-${{ github.event.pull_request.number }}.peterportal.org
+          message: Deployed staging instance to https://staging-${{ github.event.pull_request.number }}.peterportal.org Please note that it usually takes a couple minutes for the instance to be deployed and the link above may temporarily display an error message.
```

Comment:
Great work! A minor suggestion, were you able to look into potentially adding a newline between the previous message and the new one you added?